lib/tasks/sandbox_data.rake

```diff
@@ -70,3 +70,14 @@ def create_school_and_associations(lead_provider, cohort, index)
 
   school
 end
+
+def random_trn
+  return if [true, false].sample
+
+  sprintf("%07i", Random.random_number(9_999_999))
+end
+
+def create_teacher
+  mentor = User.create!(full_name: Faker::Name.name, email: Faker::Internet.email)
+  TeacherProfile.create!(user: mentor, trn: random_trn)
```

Comment:
Does this create a teacher profile with a `trn: <random number>` or a `trn: true`?

Reply:
Yes